frontend/src/metabase/dashboard/actions/cards-typed.ts

```diff
@@ -168,7 +208,10 @@ export const replaceCard =
     const dashboardId = getDashboardId(getState());
 
     let dashcard = getDashCardById(getState(), dashcardId);
-    if (isVirtualDashCard(dashcard)) {
+    if (
```

Comment:
Are there cases why we need this if statement at all?

Reply:
Good point, I added it in the first place because the action wasn't meant to be used with virtual cards. But long term it would most likely just cause confusion. Removed in [`ea9309e`](https://github.com/metabase/metabase/pull/38463/commits/ea9309ec6e83896c162bec697b700961c26c0a20)

(we're still using the "replace" action only to dashcards with questions or placeholder cards)